src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -305,29 +309,75 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     * @param {object} error: Object containing errors
     */
     handleSingleGiftErrors(error) {
-        const hasTokenizationError = error.error;
-        if (hasTokenizationError) {
-            // TODO: Handle tokenization errors
-            let tokenizationErrors = error.error;
-            if (typeof error.error === 'object') {
-                tokenizationErrors =
-                    Object.keys(tokenizationErrors).map(e => {
-                        return tokenizationErrors[e];
-                    }).join(', ');
-            }
-            return;
+        this.loadingText = null;
+
+        const hasHttpRequestError = error instanceof HttpRequestError;
+        if (hasHttpRequestError) {
+            return this.catchHttpRequestError(error);
+        }
+
+        const cardHasBeenChargedErrorButBDIFailed = error instanceof CardChargedBDIError;
+        if (cardHasBeenChargedErrorButBDIFailed) {
+            return this.catchCardChargedBDIFailedError(error);
         }
 
         const hasAuraErrors = error.body && error.body.message;
         if (hasAuraErrors) {
-            this.handleCatchOnSave(error);
-            return;
+            return this.handleCatchOnSave(error);
         }
 
         // Handle any other error
         handleError(error);
     }
 
+    catchHttpRequestError(error) {
+        this.hasPurchaseCallTimedout = error.statusCode === 408;
```

Comment:
Could we use a constant for 408? I believe we are using constants for 201 and 400.